docs/connect/grafana.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: MindsDB and Grafana
+sidebarTitle: Grafana
+---
+
+[Grafana](https://grafana.com/) is an open-source analytics and interactive visualization web application
+that allows users to ingest data from various sources, query this data, and display it on customizable charts for easy analysis. 
+
+## Data Source Grafana Setup
+
+First, Go to the `Connections` section click on `Add new connection` and Choose the `MySQL` plugin
+, as shown below.
+
+<p align="center">
+  <img src="/assets/sql/grafana_1.png" />
+</p>
+
+Now it's time to fill in the connection details.
+
+<p align="center">
+  <img src="/assets/sql/grafana_2.png" />
+</p>
+
+There are three options, as below.
+
+<Tabs>
+  <Tab title="Local MindsDB">
+    You can connect to your local MindsDB. To do that, please use the connection details below:
+    
+    ```
+    Host: `127.0.0.1:47335`
+    Username: `mindsdb`
+    Password: <leave it empty>
+    Database: <leave it empty>
+    ```
+  </Tab>
+  <Tab title="MindsDB Cloud">
+    You can connect to your MindsDB Cloud account. To do that, please use the connection details below:
+
+    ```
+    Host: cloud.mindsdb.com:3306
+    Username: <your MindsDB Cloud username>
+    Password: <your MindsDB Cloud password>
+    Database: <leave it empty>
+    ```
+
+  </Tab>
+  <Tab title="MindsDB Pro">
+    You can connect to your MindsDB Pro account. To do that, please use the connection details below:
+
+    ```
+    Host: <dedicated instace ip>:3306
+    Username: <your MindsDB Cloud username>
+    Password: <your MindsDB Cloud password>
+    Database: <leave it empty>
+    ```
+
+  </Tab>
+</Tabs>
+
+Now we are ready to Save & test the connection.
+
+## Testing the Connection
+
+Click on the `Save & test` button to check if all the provided data
+allows you to connect to MindsDB.
+
+On success, you should see the message, as below.
+
+<p align="center">
+  <img src="/assets/sql/grafana_3.png" />
+</p>
+
+## Let's Run Some Queries 
```

Comment:
Let's call this section `## Examples`

Then, add two subsections:
1. `### Querying` should include querying for full databases
2. `### Visual Query Builder` should include its content as is

Reply:
done